src/base/testcase.py

```diff
@@ -3,29 +3,6 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-import logging
-
-import pytest
-
-from src.core.api.finder.finder import highlight, wait, wait_vanish, find, find_all, exists
-from src.core.api.finder.pattern import Pattern
-from src.core.api.keyboard.key import Key, KeyModifier
-from src.core.api.keyboard.keyboard import type, key_down, key_up
-from src.core.api.mouse.mouse import *
-from src.core.api.mouse.mouse_controller import Mouse
-from src.core.api.screen.region import Region
-from src.core.api.screen.screen import *
-from src.core.api.enums import *
-from src.core.api.errors import *
-from src.core.api.location import Location
-from src.core.api.os_helpers import OSHelper
-from src.core.api.rectangle import Rectangle
-from src.core.api.settings import Settings
-from src.core.util.region_utils import RegionUtils
-from src.core.util.path_manager import PathManager
-from funcy import compose
-
-
```

Comment:
These imports cannot be deleted. I understand why the linter has flagged then, due to non-use in this module, but we must import them here in the base test case class so that subclassed test cases don't have to import the Iris API themselves.